src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -174,3 +174,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
     with self.argument_context('ams live event create') as c:
         c.argument('streaming_protocol', arg_type=get_enum_type(LiveEventInputProtocol),
                    help='The streaming protocol for the live event.')
+        c.argument('auto_start', action='store_true', help='The flag indicates if auto start the live event.')
```

Comment:
I would change the message to something like 'Start the live event automatically after creating it' (this is because I saw other flag params have something like that)